### PR TITLE
[SDL-0188] Tests for OnHashChange

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
ATF Test Scripts to check #(FORDTCN-7217)[https://adc.luxoft.com/jira/browse/FORDTCN-7217]

This PR is **[ready / not ready]** for review.

### Summary
Tests for OnHashChange notification during processing of subsription to interior vehicle data

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Resumption/InteriorVehicleData/OnHashChange/001_OnHashChange_by_module_subscription_without_moduleId.lua [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+-- Description: SDL sends the notification after successful subscription to interior vehicle data with default moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is not subscribed to modules
+--
+-- Sequence:
+-- 1. GetInteriorVD(subscribe=true, module_1) is requested
+-- SDL does:
```
Comment (VjKlepikov) [inline test_scripts/Resumption/InteriorVehicleData/OnHashChange/001_OnHashChange_by_module_subscription_without_moduleId.lua:13]:
```suggestion
-- SDL does:
send GetInteriorVD(subscribe = true, module_1) request to HMI
```
I propose to add a step. This comment relates to next scripts too.

File: test_scripts/Resumption/InteriorVehicleData/commonResumptionsInteriorVD.lua [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+function m.preconditions(isPreloadedUpdate, pCountOfRCApps)
+  if isPreloadedUpdate == nil then isPreloadedUpdate = true end
+  actions.preconditions()
+  if isPreloadedUpdate == true then
+    updatePreloadedPT(pCountOfRCApps)
+  end
+end
+
+return m
```
Comment (VjKlepikov) [inline test_scripts/Resumption/InteriorVehicleData/commonResumptionsInteriorVD.lua:130]: I propose to add a test set to this PR

**Comment by GetmanetsIrina** [comment]
All comments are processed in 9768b57

File: test_scripts/Resumption/InteriorVehicleData/commonResumptionsInteriorVD.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/Resumption/InteriorVehicleData/commonResumptionsInteriorVD.lua:130]: test set is added in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/54

**Pull request merged**